packages/expo-updates/src/Updates.ts

```diff
@@ -169,6 +170,42 @@ export async function checkForUpdateAsync(): Promise<UpdateCheckResult> {
   return result;
 }
 
+// @docsMissing
+/**
+ * Retrieves the most recent expo-updates log entries from the client, going back maxAge seconds
+ * (default: 3600 seconds = 1 hour)
```

Comment:
```suggestion
 * Retrieves the most recent expo-updates log entries from the client.
 * @param maxAge Sets the max age of retrieved log entries in seconds. Default to 3600 seconds (1 hour).
```

Method params should be documented separately.

Reply:
Fixed